portal_share_custom_filter/security/group_portal_security.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="ir_filters_auction_user_rule" model="ir.rule">
+        <field name="name">ir.filter: Auction User</field>
+        <field name="model_id" ref="base.model_ir_filters" />
+        <field
+            name="domain_force"
+        >['|',('user_id', '=', user.id),('portal_share','=',True)]</field>
+        <field
+            name="groups"
+            eval="[Command.link(ref('base_auction.group_auction_user'))]"
+        />
+        <field name="perm_unlink" eval="False" />
```

Comment:
name="groups" line is my bad. so I updated this line.

perm_unlink line is the reason why I changed from portal to action user.
If I override the portal rules, portal user will have True with perm_unlink . This does not affect standard filters. However, if you share favorites to the portal user, they could be deleted by the portal user.
<img width="919" alt="2023-01-12_19h00_14" src="https://user-images.githubusercontent.com/101688144/212040706-840a062e-ca98-4281-a7a8-c2b138eabb7f.png">
Because I set pram_unlink is False in the action user rule so that Favorite cannot be deleted.

Reply:
@kanda999 Can you please add that explanation so that the context is clear.